src/Widget/ComponentStyleSelect.php

```diff
@@ -93,12 +96,37 @@ public function generate()
                 $arrFieldOptions[] = array('value'=>'', 'label'=>'-');
             }
 
+            if($blnInvert) {
```

Comment:
@c4y 

I do understand this part and that this inverts the major options.

**However, by doing this:**
- you still have an opt-in for the extended groups
- you completely disable the skip for specific fields within L123 / L136 / L153
- you did not consider ``$objStyleGroup->extendContentElement`` at all

**From what I've tested:**
You still have to opt-in the whole group to enable the StyleManager-Widget and you can opt-out specific content-elements. You however, can not opt-out from the following due to this part of the code:

- Layout
- Page
- Article
- Form
- Module
- News
- Events

You should edit L115 (as clearly stated in the comment you wrote yourself) and rewrite the following if-statements for a proper opt-out again.